sip_server.go

```diff
@@ -6,37 +6,34 @@ import (
-	LiveShare           int            `json:"liveShare,omitempty"`
-	ModifiedBy          string         `json:"modifiedBy,omitempty"`
-	ModifiedDate        string         `json:"modifiedDate,omitempty"`
-	SavedEvent          int            `json:"savedEvent,omitempty"`
-	SharedWithAccounts  []AccountGroup `json:"sharedWithAccounts,omitempty"`
-	TestID              int            `json:"testId,omitempty"`
-	TestName            string         `json:"testName,omitempty"`
-	Type                string         `json:"type,omitempty"`
-	Interval            int            `json:"interval,omitempty"`
-	BgpMeasurements     int            `json:"bgpMeasurements,omitempty"`
-	MtuMeasurements     int            `json:"mtuMeasurements,omitempty"`
-	NetworkMeasurements int            `json:"networkMeasurements,omitempty"`
-	Port                int            `json:"port,omitempty"`
-	Protocol            string         `json:"protocol,omitempty"`
-	User                string         `json:"user,omitempty"`
-	Password            string         `json:"password,omitempty"`
-	AuthUser            string         `json:"authUser,omitempty"`
-	OptionsRegex        string         `json:"options_regex,omitempty"`
-	RegisterEnabled     int            `json:"registerEnabled,omitempty"`
-	SIPRegistrar        string         `json:"sipRegistrar,omitempty"`
-	SIPTargetTime       int            `json:"sipTargetTime,omitempty"`
-	SIPTimeLimit        int            `json:"sipTimeLimit,omitempty"`
+	Agents               []Agent        `json:"agents,omitempty"`
+	AlertsEnabled        int            `json:"alertsEnabled,omitempty"`
+	AlertRules           []AlertRule    `json:"alertRules,omitempty"`
+	APILinks             APILinks       `json:"apiLinks,omitempty"`
+	CreatedBy            string         `json:"createdBy,omitempty"`
+	CreatedDate          string         `json:"createdDate,omitempty"`
+	Description          string         `json:"description,omitempty"`
+	Enabled              int            `json:"enabled,omitempty"`
+	Groups               []GroupLabel   `json:"groups,omitempty"`
+	LiveShare            int            `json:"liveShare,omitempty"`
+	ModifiedBy           string         `json:"modifiedBy,omitempty"`
+	ModifiedDate         string         `json:"modifiedDate,omitempty"`
+	SavedEvent           int            `json:"savedEvent,omitempty"`
+	SharedWithAccounts   []AccountGroup `json:"sharedWithAccounts,omitempty"`
+	TestID               int            `json:"testId,omitempty"`
+	TestName             string         `json:"testName,omitempty"`
+	Type                 string         `json:"type,omitempty"`
+	Interval             int            `json:"interval,omitempty"`
+	BgpMeasurements      int            `json:"bgpMeasurements,omitempty"`
+	MtuMeasurements      int            `json:"mtuMeasurements,omitempty"`
+	NetworkMeasurements  int            `json:"networkMeasurements,omitempty"`
+	TargetSipCredentials SIPAuthData    `json:"targetSipCredentials,omitempty"`
```

Comment:
sip doesnt have these fields?

Reply:
The SIP server test does include BgpMeasurements, MtuMeasurements, and NetworkMeasurements as specified in the documentation.

Where this differs from the documentation is that some of the other fields listed in the documentation (authUser, password, port, protocol, and sipRegistrar) must actually exist as keys inside of a TargetSipCredentials object.  This  turned out to how TargetSipCredentials is used in voice-call, excepting that there are fewer required sub-keys when  used for sip-server.

Reply:
It was the `TargetSipCredentials` i was referring to and that sounds all good :)